data/human/culture conversations.txt

```diff
@@ -468,3 +468,55 @@ mission "Masquerade Hanging"
 			`	Moments later the small crowd allows him to depart opposite your viewpoint, without incident and without explanation. You aren't quite sure what to make of this scene as the bodies are swiftly dragged away.`
 				decline
 
+mission "News Reporter - Paradise"
+	minor
+	source
+		government "Republic"
+		attributes "paradise"
+		not attributes "station"
+	to offer
+		random < 8
+		has "event: war begins"
+		not "event: joined the free worlds"
+	on offer
+		conversation
+			`You are passing through the spaceport when you see a young woman that is walking in your direction. You realize she is a news reporter based on th multitude of camera drones trailing behind her.`
```

Comment:
Changed phrasing.
```suggestion
			`You are passing through the spaceport when you see a young woman walking in your direction. Trailing behind her is a multitude of camera drones, marking her as a news reporter.`
```

Reply:
I'm of the camp that shows the reader that they're a news reporter rather than tell. Here, a multitude of camera drones is already an oddity. I'd say that having them marked as news drones or in a colour that doesn't reflect light would be good.

Reply:
Updated my suggestion.